python/cucim/src/cucim/skimage/_shared/utils.py

```diff
@@ -638,8 +638,7 @@ def check_random_state(seed):
     if isinstance(seed, cp.random.RandomState):
         return seed
     raise ValueError(
-        "%r cannot be used to seed a numpy.random.RandomState"
-        " instance" % seed
+        "%r cannot be used to seed a cupy.random.RandomState" " instance" % seed
```

Comment:
Think we can combine the 2 strings. Also how about using f-strings?

```suggestion
        f"{seed} cannot be used to seed a cupy.random.RandomState instance"
```

Reply:
yeah, that is definitely better. not sure how this got formatted that way...

Reply:
Probably `black` did something weird